src/umpire/Umpire.hpp

```diff
@@ -122,6 +122,21 @@ bool pointer_overlaps(void* left, void* right);
  */
 bool pointer_contains(void* left, void* right);
 
+/*!
+ * \brief Check whether or not an Allocator is accessible from a given platform.
+ *
+ *\param camp::Platform p  
+ *\param umpire::Allocator a 
+ */
+bool isAccessible(Platform p, Allocator a);
```

Comment:
```suggestion
bool is_accessible(Platform p, Allocator a);
```

Use snake_case for free functions.

Reply:
This is a good catch! I didn't see that. (And I think odd that it didn't give me a warning about it)